src/app/core/components/navigation/navigation.component.ts

```diff
@@ -0,0 +1,58 @@
+import { CommonModule } from '@angular/common';
+import { Component, OnDestroy } from '@angular/core';
+import { NavigationEnd, Router } from '@angular/router';
+import { ActiveRouteNavigationEnum } from '@core/enums/active-route-nav.enum';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'navigation',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './navigation.component.html',
+  styleUrl: './navigation.component.scss'
+})
+export class NavigationComponent implements OnDestroy {
+  activeRouteNavigationEnum = ActiveRouteNavigationEnum;
+
+  activeTab: ActiveRouteNavigationEnum = ActiveRouteNavigationEnum.CharactersRoute;
+
+  routerSub!: Subscription;
+
+  constructor(private router: Router) {
+    this.routerSub = this.router.events.subscribe(event => {
+      if (event instanceof NavigationEnd) {
```

Comment:
better to use `.pipe(filter)` to check NavigationEnd

Reply:
changed